src/js/media-time-range.js

```diff
@@ -63,14 +68,22 @@ template.innerHTML = /*html*/`
       opacity: 0;
     }
 
-    :host([${MediaUIAttributes.MEDIA_PREVIEW_IMAGE}]:hover) [part~="preview-box"],
-    :host([${MediaUIAttributes.MEDIA_PREVIEW_TIME}]:hover) [part~="preview-box"] {
+    :host(:is([${MediaUIAttributes.MEDIA_PREVIEW_IMAGE}], [${MediaUIAttributes.MEDIA_PREVIEW_TIME}])[dragging]) [part~="preview-box"] {
       transition-duration: var(--media-preview-transition-duration-in, .5s);
       transition-delay: var(--media-preview-transition-delay-in, .25s);
       visibility: visible;
       opacity: 1;
     }
 
+    @media (hover: hover) {
```

Comment:
on devices that don't support hover, don't add the :hover selector.
:hover gets enabled by tap on mobile and is sticky. use the explicit [dragging] attribute...